e2e/with-php-level/src/SomeClass.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+final class SomeClass
+{
+    public function __construct(private readonly string $b)
```

Comment:
this not transformed to readonly class even with `PHP_INT_MAX` on level, as I define 

```php
->withPhpVersion(PhpVersionFeature::ATTRIBUTES)
```

on rector.php e2e test https://github.com/rectorphp/rector-src/pull/6384/files#diff-464b8708ccfbbbeecb99d3dc19bea3998a6d55af99232759e70edae715f3abf0 so it only apply up to php 8.0 feature.